src/components/LeftSidebar/ConversationsList/Conversation.vue

```diff
@@ -21,6 +21,7 @@
 
 <template>
 	<ListItem :title="item.displayName"
+		:titleTooltip="{content: item.displayName, boundariesElement: boundariesElement}"
```

Comment:
```suggestion
		:titleTooltip="{content: item.displayName.length > 20 ? item.displayName : '', boundariesElement: boundariesElement}"
```
Should the tooltip display for any length name (follows the standard of the spreed right sidebar) or only for names above a certain character length (follows the standard of [server user list](https://github.com/nextcloud/server/blob/e4378fd18cb39407097f40431cd28c1fa01ba75d/apps/settings/src/components/UserList/UserRowSimple.vue#L17))?